src/components/table/table-column-resize/resizable-table-column.component.ts

```diff
@@ -102,9 +102,26 @@ export class ResizableTableColumnComponent implements AfterViewInit, OnDestroy {
     /** Emit when all observables should be unsubscribed */
     private _onDestroy = new Subject<void>();
 
+    resizeObservable$: Observable<Event>;
+
     constructor(private _elementRef: ElementRef, @Inject(RESIZABLE_TABLE_SERVICE_TOKEN) private _table: BaseResizableTableService, private _renderer: Renderer2) { }
 
     ngAfterViewInit(): void {
+        this.resizeObservable$ = fromEvent(window, 'resize');
+
+        // ensure the correct width gets emitted when the window size changes
+        this.resizeObservable$.pipe(takeUntil(this._onDestroy))
+            .subscribe(() => {
+                this.setColumnWidth();
+                this.setColumnFlex();
+
+                const width = this._table.getColumnWidth(this.getCellIndex(), ColumnUnit.Pixel);
+
+                if (!isNaN(width)) {
+                    this.widthChange.emit(width);
+                }
+            });
+
```

Comment:
There is an `onResize` subscription below, which looks like it is not getting fired, I suggest trying to fix it instead as that should resolve the issue. We can't just rely on window resize to do this as a table could be resized for many reasons, e.g. a panel opening shrinking the available space. This window resize event would not get triggered in that case so this would not fix the issue for that case.